package.json

```diff
@@ -40,45 +36,45 @@
 		"docs:serve": "./node_modules/.bin/http-server docs/jsdoc/",
 		"precommit": "lint-staged",
 		"check:dependencies": "./node_modules/.bin/snyk protect",
-		"prepush": "./node_modules/.bin/eslint $(git ls-files '*.js') && npm run check:dependencies",
+		"prepush":
+			"./node_modules/.bin/eslint $(git ls-files '*.js') && npm run check:dependencies",
 		"prepublishOnly": "npm run check:dependencies"
 	},
 	"dependencies": {
-		"async": "=2.4.1",
+		"async": "=2.6.1",
 		"bignumber.js": "=7.2.1",
-		"bluebird": "=3.5.1",
-		"body-parser": "=1.18.2",
+		"bluebird": "=3.5.3",
+		"body-parser": "=1.18.3",
 		"bytebuffer": "=5.0.1",
-		"change-case": "=3.0.1",
-		"colors": "=1.1.2",
-		"commander": "=2.9.0",
-		"compression": "=1.7.1",
-		"cors": "=2.8.3",
+		"change-case": "=3.0.2",
+		"colors": "=1.3.2",
+		"commander": "=2.19.0",
+		"compression": "=1.7.3",
+		"cors": "=2.8.5",
 		"debug": "=3.1.0",
-		"deep-diff": "=1.0.1",
-		"ejs": "=2.5.6",
-		"express": "=4.16.0",
+		"deep-diff": "=1.0.2",
+		"express": "=4.16.4",
 		"express-domain-middleware": "=0.1.0",
 		"express-query-int": "=1.0.1",
 		"express-rate-limit": "=2.8.0",
-		"extend": "=3.0.1",
-		"find": "=0.2.7",
+		"extend": "=3.0.2",
+		"find": "=0.2.9",
 		"fs-extra": "=5.0.0",
 		"ip": "=1.1.5",
-		"js-yaml": "=3.10.0",
+		"js-yaml": "=3.12.0",
 		"json-refs": "=3.0.12",
 		"lisk-elements": "=1.0.0",
 		"lodash": "=4.17.11",
 		"lisk-newrelic": "LiskHQ/lisk-newrelic#1ea09b4",
 		"method-override": "=2.3.10",
-		"newrelic": "=4.8.1",
-		"pg-monitor": "=0.9.0",
+		"newrelic": "=4.10.0",
+		"pg-monitor": "=1.1.0",
 		"pg-promise": "=8.2.1",
```

Comment:
Please update this package as well.

Reply:
@nazarhussain could you please check if this comment can be resolved?

Reply:
@nazarhussain @MaciejBaj updated pg-promise to latest minor version (changelog did mention a breaking change but for a feature we're not using)